Makefile

```
@@ -420,3 +420,7 @@ generate-images:
 					$(TMPDIR)/images/64.png $(TMPDIR)/images/128.png \
 					$(PWD)/public/img/favicon.ico
 	rm -rf $(TMPDIR)/images
+	
+.PHONY: pr
+pr:
+    $(GO) run contrib/pr/checkout.go $(PR)
```

Regarding failed CI, I think Makefile is just complaining about this not being a tab.
```suggestion
	$(GO) run contrib/pr/checkout.go $(PR)
```

---

Yes, I edit it via github web interface and it fail ... I was waiting to have time to fix this but thanks for the correction.

---

Hah sorry, I'm just really excited to be using this. 😄